packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/softphone.controller.js

```diff
@@ -5,20 +5,38 @@ export default class {
     this.softphoneService = SoftphoneService;
   }
 
-  handleToggleSwitch() {
+  handleErrorOnSwitch() {
+    this.error_option = true;
```

Comment:
To be consistent with camelCase used elsewhere
```suggestion
    this.errorOption = true;
```